kyuubi-server/src/test/scala/org/apache/kyuubi/server/rest/client/AdminRestApiSuite.scala

```diff
@@ -84,4 +87,64 @@ class AdminRestApiSuite extends RestClientTestHelper {
     engines = adminRestApi.listEngines("spark_sql", "user", "default", "").asScala
     assert(engines.size == 0)
   }
+
+  test("list/close session") {
+    fe.be.sessionManager.openSession(
+      TProtocolVersion.findByValue(1),
```

Comment:
Looks better to keep same with below test' usage `HIVE_CLI_SERVICE_PROTOCOL_V2`

Reply:
thanks. fixed